ui/app/conf-tx.js

```diff
@@ -48,7 +49,7 @@ ConfirmTxScreen.prototype.render = function () {
   var isNotification = isPopupOrNotification() === 'notification'
 
   log.info(`rendering a combined ${unconfTxList.length} unconf msg & txs`)
-  if (unconfTxList.length === 0) return null
+  if (unconfTxList.length === 0) return h(Loading)
```

Comment:
This doesn't seem to work right, though.  Loading is slow, and this view is often uncomfortably empty.  The reasons though, are deeply tangled.